image/tarexport/load.go

```diff
@@ -422,8 +423,8 @@ func checkCompatibleOS(imageOS string) error {
 		return fmt.Errorf("cannot load %s image on %s", imageOS, runtime.GOOS)
 	}
 	// Finally, check the image OS is supported for the platform.
```

Comment:
Comment no longer seems to match the actual check (also see below)

Reply:
This comment will be removed in the update